projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -258,18 +258,6 @@ public static function load_coming_soon() {
 			require_once ABSPATH . 'wp-admin/includes/plugin.php';
 		}
 
-		/**
-		 * Explicitly pass $markup = false in get_plugin_data to avoid indirectly calling wptexturize that could cause unintended side effects.
-		 * See: https://developer.wordpress.org/reference/functions/get_plugin_data/
-		 */
-		$invalid_fse_version_active =
-			is_plugin_active( 'full-site-editing/full-site-editing-plugin.php' ) &&
-			version_compare( get_plugin_data( WP_PLUGIN_DIR . '/full-site-editing/full-site-editing-plugin.php', false )['Version'], '3.56084', '<' );
```

Comment:
Hmmm.... could it have actually broken sites that "On WoA sites, users may be using non-symlinked older versions of the FSE plugin."

ref: 

https://github.com/Automattic/jetpack/pull/28932/files#diff-d5a84593bf9ba4c065049a4ecf67ee10ee2b3840eb67825a8646fe1c94b1ae50R49